lib/versioning/node/index.spec.ts

```diff
@@ -1,5 +1,5 @@
 import { DateTime } from 'luxon';
-import { isStable, isValid, api as nodever } from '.';
+import { isStable, isValid, api as nodever, valueToVersion } from '.';
```

Comment:
The new feature is not (yet) documented.

I think our `docs/usage/node.md` [^source] file is a good place to cover the Node.js codename support in Renovate.

[^source]: https://github.com/renovatebot/renovate/blob/main/docs/usage/node.md

Reply:
I checked the box and forgot to push 🤦 PTAL at https://github.com/renovatebot/renovate/pull/13466/commits/1a638fb033b08efdefc2d089de3f08fbad4400d8 😀